pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -292,75 +302,20 @@ func GetDomainSpecWithFlags(dom cli.VirDomain, flags libvirt.DomainXMLFlags) (*a
 	return domain, nil
 }
 
-func (l CloudHypervisorLibvirtWrapper) StartLibvirtd(stopChan chan struct{}) {
-	// we spawn libvirtd from virt-launcher in order to ensure the libvirtd process
-	// doesn't exit until virt-launcher is ready for it to. Virt-launcher traps signals
-	// to perform special shutdown logic. These processes need to live in the same
-	// container.
-
-	go func() {
-		for {
-			exitChan := make(chan struct{})
-			args := []string{"-f", "/etc/libvirt/libvirtd.conf"}
-			cmd := exec.Command("/libvirt/build/src/libvirtd", args...)
```

Comment:
Out of curiosity, isn't libvirtd not needed in the new approach?

Reply:
Libvirtd is not needed in the proposed approach, because its function will be carried out by `virtchd` - the modular libvirt daemon for cloud-hypervisor.

Reply:
Sounds good! Thanks for clarifying!